packages/flutter_tools/test/general.shard/web/bootstrap_test.dart

```diff
@@ -14,9 +14,11 @@ void main() {
       mapperUrl: 'mapper.js',
     );
     // require js source is interpolated correctly.
-    expect(result, contains('requireEl.src = "require.js";'));
+    expect(result, contains('"requireJs": "require.js"'));
+    expect(result, contains('requireEl.src = getTTScriptUrl("requireJs");'));
     // stack trace mapper source is interpolated correctly.
-    expect(result, contains('mapperEl.src = "mapper.js";'));
+    expect(result, contains('"mapper": "mapper.js"'));
+    expect(result, contains('mapperEl.src = getTTScriptUrl("mapper");'));
```

Comment:
Are these tests even useful? We can generate bad code then test that we generated bad code :)

Reply:
Indeed, I'm just following the pattern that was left there before. We might need to revisit this.